samples/snippets/test_ca_pools.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import typing
+import uuid
+
+import google.auth
+
+from create_ca_pool import create_ca_pool
+from delete_ca_pool import delete_ca_pool
+from list_ca_pools import list_ca_pools
+
+PROJECT = google.auth.default()[1]
+LOCATION = "europe-west1"
+
+
+def generate_name() -> str:
+    return "i" + uuid.uuid4().hex[:10]
+
+
+def test_create_ca_pool(capsys: typing.Any) -> None:
+    CA_POOL_NAME = generate_name()
+
+    create_ca_pool(PROJECT, LOCATION, CA_POOL_NAME)
+
+    out, _ = capsys.readouterr()
+
+    assert re.search(
+        f'Operation result: name: "projects/{PROJECT}/locations/{LOCATION}/caPools/{CA_POOL_NAME}"',
+        out,
+    )
+
+    delete_ca_pool(PROJECT, LOCATION, CA_POOL_NAME)
```

Comment:
Please use pytest fixtures for resources like `ca_pool` so that they are torn down even if the test fails. See [this example](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/927a5641395093a7a9b388ce7516d030d4c07e18/storage/cloud-client/bucket_lock_test.py#L40-L49) and https://docs.pytest.org/en/6.2.x/fixture.html